server/zoom/agent/entities/application.py

```diff
@@ -558,13 +558,38 @@ def _init_actions(self, settings):
         return actions
 
     def _determine_read_only(self, actions):
+        # Sentinel config may include either start or restart blocks, if either are disabled show as read-only
         start_action = actions.get('start', None)
+        restart_action = actions.get('restart', None)
 
-        if start_action is None:
-            self._read_only = True
-        elif start_action.disabled is True:
+        # Two special cases - both start and restart and neither
+        if start_action is not None and restart_action is not None:
+            if start_action.disabled is True and restart_action.disabled is True:
```

Comment:
https://www.python.org/dev/peps/pep-0008/#programming-recommendations
Avoid `is True`